tests/test_cli.py

```diff
@@ -166,8 +166,8 @@ def test_valid_stomp_config_for_listener(
         input="\n",
     )
     assert (
-        result.output
-        == "Subscribing to all bluesky events from localhost:61613\nPress enter to exit"
+        "Subscribing to all bluesky events from localhost:61613\nPress enter to exit"
+        in result.output
```

Comment:
Strange that you needed to do this, I can spot the change that caused it, any ideas?

Reply:
I'm presuming there's an extra trailing \n somewhere?